packages/p2p/src/pages/orders/order-details/order-details-timer.tsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { observer } from 'mobx-react-lite';
```

Comment:
```suggestion
import { observer } from '@deriv/stores';
```

we also gotta change where we import observer from 👀